ajax/libs/jimp/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jimp",
+  "filename": "jimp.min.js",
+  "version": "0.2.27",
+  "description": "An image processing library written entirely in JavaScript (i.e. zero external or native dependencies).",
+  "keywords": [
+    "image",
+    "image processing",
+    "image manipulation",
+    "png",
+    "jpg",
+    "jpeg",
+    "bmp",
+    "resize",
+    "scale",
+    "crop"
+  ],
+  "author": "Oliver Moran <oliver.moran@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oliver-moran/jimp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/oliver-moran/jimp.git",
+    "fileMap": [
+      {
+        "basePath": "",
```

Comment:
Here it is.
We automatically update the files from the repository files structure.
![image](https://cloud.githubusercontent.com/assets/8349587/24068299/476e8b72-0bc6-11e7-8859-b8f0b7c983e9.png)